editor/scene_create_dialog.cpp

```diff
@@ -131,9 +131,24 @@ void SceneCreateDialog::update_dialog() {
 	root_name = root_name_edit->get_text().strip_edges();
 	if (root_name.is_empty()) {
 		root_name = scene_name.get_file().get_basename();
```

Comment:
I hope this is the last one. Sorry for the many rounds of the review.

```suggestion
		root_name = scene_name_edit->get_text().strip_edges();
```

![](https://github.com/godotengine/godot/assets/47700418/6f56a38a-2ca5-45fb-ac1e-1e315301c311)

Reply:
Note that subfolders are not supported by this field. But it is not related to this PR.

![](https://github.com/godotengine/godot/assets/47700418/8de6d09b-dd91-4a85-ad61-9b4b5c6e6a93)